tests/XrdHttpTests/XrdHttpTests.cc

```diff
@@ -628,4 +629,25 @@ TEST(XrdHttpTests, parseReprDigest) {
     XrdHttpHeaderUtils::parseReprDigest(input,output);
     ASSERT_EQ(expectedMap, output);
   }
+}
+
+TEST(XrdHttpTests, getCORSAllowOriginHeader) {
+  std::unordered_set<std::string> allowedOrigins = {
+    "https://helloworld.cern.ch/"
+    "https://anotherorigins.cern.ch/"
```

Comment:
By construction I don't think the origin header can have the trailing slash, the allowed syntax is:

```
Origin: null
Origin: <scheme>://<hostname>
Origin: <scheme>://<hostname>:<port>
```

Reply:
Similarly I think the example in the PR description is invalid.